### DISCUSSION TITLE
I can't flash it Zigbee Koenkk 20240710 для Sonoff Zigbee ZBPro

### DISCUSSION THREAD

**Discussion by timurlan75** [arendst/Tasmota#22257] [Support request] [answered]
I can't flash it Zigbee Koenkk 20240710 для Sonoff Zigbee ZBPro. I replaced all the files for berry (cc2652_flasher.be, intelhex.be, sonoff_zb_pro_flasher.be) and it gives an error:
FLH: Starting verification of HEX file
BRY: Exception> 'index_error' - string index out of range
stack traceback:

: in function parse

: in function 'check`

: in function main

**Comment by s-hadinger** [comment]
Can you describe what commands you used and what files. Ideally show a screenshot of your webui showing the filesystem

**Reply by timurlan75** [reply]
files taken from here https://github.com/s-hadinger/Tasmota/tree/zigbee_fix_flasher_len_4/tasmota/berry/zigbee

I entered the following command :
import sonoff_zb_pro_flasher as cc
cc.load("SonoffZBPro_coord_20240710.hex")
cc.check()

**Comment by timurlan75** [comment]
![Screenshot_20241010-020451](https://github.com/user-attachments/assets/4891a720-7377-4c93-8586-1fb4b8b10970)

**Comment by s-hadinger** [comment, marked as answer]
The file size of the firmware is completely wrong. It should be 506662 bytes. Did you use "Download raw file" in Github?

**Reply by timurlan75** [reply]
It seems the device has been flashed. Thanks for your help.
